src/main/java/com/vkatit/cinema/service/PDFEditor.java

```diff
@@ -0,0 +1,160 @@
+package com.vkatit.cinema.service;
+
+
+import com.itextpdf.io.font.constants.StandardFonts;
+import com.itextpdf.kernel.font.PdfFont;
+import com.itextpdf.kernel.font.PdfFontFactory;
+import com.itextpdf.kernel.geom.PageSize;
+import com.itextpdf.kernel.pdf.PdfDocument;
+import com.itextpdf.kernel.pdf.PdfPage;
+import com.itextpdf.kernel.pdf.PdfReader;
+import com.itextpdf.kernel.pdf.PdfWriter;
+import com.itextpdf.layout.Canvas;
+import com.itextpdf.layout.Document;
+import com.itextpdf.layout.element.Paragraph;
+import com.itextpdf.layout.element.Text;
+import com.itextpdf.layout.properties.TextAlignment;
+import com.itextpdf.layout.properties.VerticalAlignment;
+import com.vkatit.cinema.model.Printable;
+import com.vkatit.cinema.model.Ticket;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PDFEditor implements Printable {
+    @Value("${rawTicketPath}")
+    private String rawTicketPath;
+    @Value("${generatedTicketPath}")
+    private String outputFilePath;
+
+    public PDFEditor(@Value("${rawTicketPath}") String rawTicketPath,
+                     @Value("${generatedTicketPath}") String outputFilePath) {
+        this.rawTicketPath = rawTicketPath;
+        this.outputFilePath = outputFilePath;
+    }
+
+    @Override
+    public byte[] getTicket(Ticket ticket) {
+        return handleExceptionsAndReturnBytesArray(ticket);
+    }
+
+    @ExceptionHandler
+    public byte[] handleExceptionsAndReturnBytesArray(Ticket ticket) {
+        try {
```

Comment:
code pullition. 2 try catched with exactly the same behavior. and after it's even rethrown. so try/catch make 0 sense.